sympy/combinatorics/fp_groups.py

```diff
@@ -125,6 +133,8 @@ def __init__(self, fp_grp, subgroup):
         self.p = [0]
         self.A = list(chain.from_iterable((gen, gen**-1) \
                 for gen in self.fp_group.generators))
+        # the mathematical coset table which represented using the list of
```

Comment:
'coset table which is represented ...', or maybe just 'coset table as a list of lists'.